.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
https://github.com/MadaShindeInai/tasks/blob/master/react/modules/module01/configs.md

configs should be the same (should contain **the same rules**).

Reply:
"rules": {
    "react-hooks/rules-of-hooks": "error",
    "react-hooks/exhaustive-deps": "warn",
    "comma-dangle": ["error", "only-multiline"],
    "react/prop-types": "off",
    "react/display-name": "off",
    "@typescript-eslint/explicit-function-return-type": "off",
    "prettier/prettier": ["error", { "endOfLine": "auto" }],
    "@typescript-eslint/interface-name-prefix": "off",
    "@typescript-eslint/ban-ts-comment": "error",
    "@typescript-eslint/explicit-module-boundary-types": "off",
    "@typescript-eslint/no-empty-function": "off",
    "@typescript-eslint/no-explicit-any": "error",
    "@typescript-eslint/no-var-requires": "off",
    "react/jsx-uses-react": "off",
    "react/react-in-jsx-scope": "off"
  },

Reply:
Now eslint config is correct. React hook roles are working correctly.
![eslint-correction](https://user-images.githubusercontent.com/54704072/177563322-6d3fd527-08ce-4fb8-b3cd-261d4a01e35f.PNG)